openfecwebapp/views.py

```diff
@@ -46,19 +37,75 @@ def render_table(data_type, results, params):
 
 type_map = {
     'candidates': lambda x: x,
-    'candidate': map_candidate_page_values,
+    'candidate': lambda x: CandidateSchema().dump(x).data,
     'committees': lambda x: x,
-    'committee': map_committee_page_values
+    'committee': lambda x: CommitteeSchema().dump(x).data
 }
 
-def render_page(data_type, c_data):
-    # not handling error at api module because sometimes its ok to 
+
+def render_page(data_type, *args, **kwargs):
```

Comment:
We accept `*args` here, but only ever use `args[0]`. The `**kwargs` splat also seems to accept `candidates` and `committees` keys. A clearer signature might be `def render_page(data_type, data, committees=None, candidates=None)`.

More broadly, the two branches this function can take don't share a lot of code, and putting them into a single function is IMO harder to read / test / debug than using two separate functions, `render_committee` and `render_candidate`.